.pre-commit-config.yaml

```diff
@@ -0,0 +1,23 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  rev: v0.1.14
+  hooks:
+    # Run the linter.
+  - id: ruff
+    # Run the formatter.
+  - id: ruff-format
+    args: [--check]
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.5.0
+  hooks:
+  - id: trailing-whitespace
+  - id: end-of-file-fixer
+  - id: check-yaml
+  - id: check-toml
+  - id: check-ast
+  - id: check-added-large-files
```

Comment:
What is considered a large file? Notebooks are sometimes huge.

Reply:
Your can specify what is "too large" with `args: ['--maxkb=123']`? (default=500kB).

[[ref](https://github.com/pre-commit/pre-commit-hooks?tab=readme-ov-file#hooks-available)]